data_model/src/lib.rs

```diff
@@ -1469,42 +1429,23 @@ impl TryFrom<f64> for Value {
     }
 }
 
-impl From<trigger::Trigger<TriggeringFilterBox, Executable>> for Value {
-    fn from(trigger: trigger::Trigger<TriggeringFilterBox, Executable>) -> Self {
-        Value::Identifiable(IdentifiableBox::Trigger(TriggerBox::Raw(trigger)))
-    }
-}
-
-impl From<trigger::Trigger<TriggeringFilterBox, trigger::OptimizedExecutable>> for Value {
-    fn from(trigger: trigger::Trigger<TriggeringFilterBox, trigger::OptimizedExecutable>) -> Self {
-        Value::Identifiable(IdentifiableBox::Trigger(TriggerBox::Optimized(trigger)))
-    }
-}
-
-impl TryFrom<Value> for trigger::Trigger<TriggeringFilterBox, Executable> {
-    type Error = ErrorTryFromEnum<Value, Self>;
-
-    fn try_from(value: Value) -> Result<Self, Self::Error> {
-        if let Value::Identifiable(IdentifiableBox::Trigger(TriggerBox::Raw(trigger))) = value {
-            return Ok(trigger);
-        }
-
-        Err(Self::Error::default())
-    }
-}
-
-impl TryFrom<Value> for trigger::Trigger<TriggeringFilterBox, trigger::OptimizedExecutable> {
-    type Error = ErrorTryFromEnum<Value, Self>;
-
-    fn try_from(value: Value) -> Result<Self, Self::Error> {
-        if let Value::Identifiable(IdentifiableBox::Trigger(TriggerBox::Optimized(trigger))) = value
-        {
-            return Ok(trigger);
-        }
-
-        Err(Self::Error::default())
-    }
-}
+// impl From<trigger::Trigger<TriggeringFilterBox>> for Value {
+//     fn from(trigger: trigger::Trigger<TriggeringFilterBox>) -> Self {
+//         Value::Identifiable(IdentifiableBox::Trigger(trigger))
+//     }
+// }
+//
+// impl TryFrom<Value> for trigger::Trigger<TriggeringFilterBox> {
+//     type Error = ErrorTryFromEnum<Value, Self>;
+//
+//     fn try_from(value: Value) -> Result<Self, Self::Error> {
+//         if let Value::Identifiable(IdentifiableBox::Trigger(trigger)) = value {
+//             return Ok(trigger);
+//         }
+//
+//         Err(Self::Error::default())
+//     }
+// }
```

Comment:
instead of commenting out everything, I would recommend you to use `unimplemented` in the body of the impl block. That way you will not accidentally forget to uncomment

Reply:
I meant to delete these altogether, they aren't needed anymore